PocketKit/Sources/PocketKit/Article/Cells/ImageComponentCell.swift

```diff
@@ -86,7 +86,8 @@ extension ImageComponentCell {
                    result.image.size.width >= idealWidth {
                     self?.imageView.backgroundColor = UIColor(.clear)
                 } else {
-                    self?.imageView.backgroundColor = UIColor(.ui.grey7)
+                    let isCaptionHidden = self?.captionTextView.isHidden ?? false
```

Comment:
This works, but I actually really like your idea of putting this in the view model.
How about we add `let imageViewBackgroundColor` (or some better name) to the view model. That way we can just say `imageView.backgroundColor = model.imageViewBackgroundColor` and put the logic in the view model?

Reply:
sounds good, will update! just to be clear, did you meant `viewModel.imageViewBackgroundColor` ?

Reply:
yes `viewModel` not `model` :)